collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -1179,10 +1194,13 @@ def populate_queries(self):
 
         if self.server_version >= 90400:
             self.queries[query_factory(QUERY_NAME_AUTOVACUUM)] = METRICS[QUERY_NAME_AUTOVACUUM]
-
+	
         if self.server_version >= 100000:
             self.queries[query_factory(QUERY_NAME_STANDBY_LAG)] = METRICS[QUERY_NAME_STANDBY_LAG]
 
+        QUERY_STAT_REPLICATION[DEFAULT] = CREATE_STAT_QUERY(self.secondaries)
```

Comment:
`QUERY_STAT_REPLICATION` is a global parameter. We shouldn't rewrite it. It could affect other data collection jobs.

Reply:
Renamed to QUERY_DB_REPLICATION.
It's a dynamic query, depending on the number of slave nodes I have to redefine the value of the variable.